create-project.py

```diff
@@ -125,3 +159,44 @@
 # Add all files and commit them
 subprocess.call(["git", "add", "-A"], cwd=project_dir)
 subprocess.call(["git", "commit", "-m", "Init with GTK Rust Template"], cwd=project_dir)
+
+items_to_delete = [
+    Path(".flatpak-builder"),
+    Path(".github"),
+    Path("build-aux"),
+    Path("data"),
+    Path("hooks"),
+    Path("po"),
+    Path("src"),
+    Path(".editorconfig"),
+    Path(".gitignore"),
+    Path(".gitlab-ci.yml"),
+    Path("Cargo.lock"),
+    Path("Cargo.toml"),
+    Path("LICENSE.md"),
+    Path("README.md"),
+    Path("create-project.py"),
+    Path("meson.build"),
+    Path("meson_options.txt"),
```

Comment:
I think Python doesn't really care if it is a `Path` or `str` so removing the `Path`s might be shorter...